### DISCUSSION TITLE
Shorten/minimize CSS class (&id) names, both in CSS & HTML with Parcel?

### DISCUSSION THREAD

**Discussion by juanmirocks** [parcel-bundler/parcel#8903] [Q&A]
I'm looking to optimize/minimize my CSS & HTML files, by renaming the CSS class & id names to very short names (e.g. `.a`), similar to what this Google PostCSS plugin does: https://github.com/google/postcss-rename

AFAIK this is not readily available in parcel yet but I assume it can easily be done via plugins.

* Are there already plugins for this? I couldn't find it.
* If I were to write my own plugin, 
  * I assume this would be an [Optimizer](https://parceljs.org/plugin-system/optimizer/) plugin?
  * Is there some relatively similar code somewhere or other pointers available?

**Comment by devongovett** [comment]
You can use postcss with Parcel - just add that plugin to a `.postcssrc` file. See https://parceljs.org/languages/css/#postcss.

That said, I'm not sure how this would work to also update your HTML/JS to use the shortened classes.

**Reply by juanmirocks** [reply]
OK. I will report back if I experiment with also updating the HTML.
